Jenkinsfile

```diff
@@ -11,6 +11,11 @@ try {
       checkout scm
       // Make sure not to delete the folder that Jenkins allocates to store scripts
       sh 'git clean -ffdx -e .????????'
+      // Use https url to work around the permission problem
+      sh 'git config --file=.gitmodules submodule.realm/realm-library/src/main/cpp/object-store.url https://github.com/realm/realm-object-store.git'
```

Comment:
You can make this work in Jenkins by having it do the SCM operations with an SSH agent active with the appropriate SSH keys loaded. It looks something like:
```groovy
sshagent(['realm-ci-ssh']) {
    // …
}
```

Though, as I mentioned above, I think it's a mistake to specify the `git@` URL in the .gitmodules file to begin with.